test/broadcast_tables/conftest.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This file configures pytest for all tests in this directory, and also
+# defines common test fixtures for all of them to use
+
+import pathlib
+import sys
+
+# Add test.pylib to the search path
```

Comment:
Is this actually needed? Do we import from `pylib` anywhere?

Reply:
You're right, I removed the `conftest.py` file.